fix: update links in README

igorline opened the pull request:
Closes #751

igorline commented on README.md:
```diff
@@ -67,7 +67,7 @@ Witnet-rust is published under the [GNU General Public License v3.0][license].
 [install]: https://docs.witnet.io/try/run-a-node/
-[Contributing]: https://docs.witnet.io/contributing
+[Contributing]: https://docs.witnet.io/community/contributing/
 [development]: https://docs.witnet.io/development
```
@aesedepece @mariocao 
don't know whom to tag here correctly, but this link seems should be replaced with `install` one?

aesedepece replied:
Why? This is only used by the link in the "Contributions are more than welcome." sentence, right?

aesedepece replied:
I mean, I think it is perfect just in the way you updated it in your commit :wink:

igorline replied:
This comment is actually related to line `71` :)

igorline replied:
![image](https://user-images.githubusercontent.com/1465430/78050277-50fb3200-737c-11ea-9a62-9d384f0e64e5.png)

aesedepece replied:
True, I think we should separate that into three different links, one for installation, another for compilation and the last one for development.

igorline replied:
Ok, will split them and post update here

igorline replied:
I was checking how we can split those.
Installation is `try/run-a-node` obviously.
We miss compilation instructions. Should they go into documentation or be present in README.md in this repo?
What should go for development?

aesedepece replied:
Maybe "development instructions" don't sense anymore, the contributing guide is already quite extensive in that sense. What do you think?

Regarding compilation instructions, [here are some for Ubuntu][ubuntu] that I believe are up to date. Just in case it helps.

[ubuntu]: https://github.com/witnet/documentation/issues/31#issue-591847283

igorline replied:
Yeah, I think contributing guide, installation/running and compilation is enough.

Already spotted those compilation instruction will be adding PR to documentation with those.
Thank you

igorline commented:
Yeah, I will check how "Contributing" page in documentation differs from doc in #1132

igorline commented:
> This is great for now. After #1132 is merged, we should reconsider whether to point to the documentation or to the new `CONTRIBUTING` file internal to the repository (provided that they have the same content indeed)

Probably contributing.md should be pointing to contributing in documentation to avoid necessity to sync both of them

tmpolaczyk commented:
Hi, can you squash all the commits into one?

igorline commented:
@tmpolaczyk, you mean commits in this PR https://github.com/witnet/documentation/pull/32?
Yeah, I can squash them on merge after approval :)

igorline commented:
Sorry it got messed up a bit.
Removed broken or otherwise currently duplicating link on installation instructions and updated link to contributing guide in documentation book.

tmpolaczyk commented:
> @tmpolaczyk, you mean commits in this PR [witnet/documentation#32](https://github.com/witnet/documentation/pull/32)?
> Yeah, I can squash them on merge after approval :)

Yes, sorry wrong repo :smile:

aesedepece commented:
Merged as 5d80dc7ca67a36951d14d20b5480052e8ca6494e
